src/OrchardCore/OrchardCore/Extensions/Features/TypeFeatureProvider.cs

```diff
@@ -6,7 +6,7 @@ namespace OrchardCore.Environment.Extensions
 {
     public class TypeFeatureProvider : ITypeFeatureProvider
     {
-        private readonly ConcurrentDictionary<Type, IFeatureInfo> _features = new();
+        private readonly ConcurrentDictionary<Type, IFeatureInfo> _features = new ConcurrentDictionary<Type, IFeatureInfo>();
```

Comment:
so sad @jtkech  :(

Reply:
@deanmarcussen it lived a very short time but it was beautiful ;)